contracts/teller-network/EscrowRelay.sol

```diff
@@ -0,0 +1,202 @@
+  }
+
+  /**
+   * @notice Set gas station network hub address
+   * @dev Only contract owner can execute this function
+   * @param _relayHub New relay hub address
+   */
+  function setRelayHubAddress(address _relayHub) public onlyOwner {
+    set_relay_hub(RelayHub(_relayHub));
+  }
+
+  /**
+   * @notice Determine if the timeout for relaying a create/cancel transaction has passed
+   * @param _account Account to verify
+   * @return bool
+   */
+  function canCreateOrCancel(address _account) public view returns(bool) {
+    return (lastActivity[_account] + 15 minutes) < block.timestamp;
+  }
+
+  /**
+   * @notice Create a new escrow
+   * @param _offerId Offer
+   * @param _tradeAmount Amount buyer is willing to trade
+   * @param _tradeType Indicates if the amount is in crypto or fiat
+   * @param _assetPrice Indicates the price of the asset in the FIAT of choice
+   * @param _statusContactCode The address of the status contact code
+   * @param _location The location on earth
+   * @param _username The username of the user
+   * @param _nonce buyer's nonce
+   * @param _signature buyer's signature
+   */
+  function create (
+    uint _offerId,
+    uint _tradeAmount,
+    uint8 _tradeType,
+    uint _assetPrice,
+    bytes memory _statusContactCode,
+    string memory _location,
+    string memory _username,
+    uint _nonce,
+    bytes memory _signature
+  ) public returns (address instance) {
+    lastActivity[get_sender()] = block.timestamp;
+    instance = factory.create(
+      _offerId,
+      _tradeAmount,
+      _tradeType,
+      _assetPrice,
+      _statusContactCode,
+      _location,
+      _username,
+      _nonce,
+      _signature
+    );
+  }
+
+  /**
+   * @notice Mark transaction as paid
+   * @param _escrow Escrow to mark as paid
+   */
+  function pay(address _escrow) public {
+    address sender = get_sender();
+    Escrow(_escrow).pay_relayed(sender);
+  }
+
+  /**
+   * @notice Cancel an escrow
+   * @param _escrow Escrow to cancel
+   */
+  function cancel(address _escrow) public {
+    address sender = get_sender();
+    lastActivity[sender] = block.timestamp;
+    Escrow(_escrow).cancel_relayed(sender);
+  }
+
+  /**
+   * @notice Open a dispute
+   * @param _escrow Escrow to open a dispute
+   * @param _motive Motive a dispute is being opened
+   */
+  function openCase(address _escrow, string memory _motive) public {
+    address sender = get_sender();
+    Escrow(_escrow).openCase_relayed(sender, _motive);
+  }
+
+  // ========================================================================
+  // Gas station network
+
+  function accept_relayed_call(
+    address relay,
+    address from,
+    bytes memory encoded_function,
+    uint gas_price,
+    uint transaction_fee
+  ) public view returns(uint32) {
+    bytes4 fSign;
+    assembly {
+      fSign := mload(add(encoded_function, add(0x20, 0)))
+    }
+
+    if(from.balance > 600000 * gas_price) return ERROR_ENOUGH_BALANCE;
```

Comment:
Did you increase this? Since creating an escrow is now more costly?

Reply:
Good catch. I did it on #327